src/main/sensors/barometer.c

```diff
@@ -160,6 +160,7 @@ void pgResetFn_barometerConfig(barometerConfig_t *barometerConfig)
 #define NUM_GROUND_LEVEL_CYCLES   10        // calibrate baro to new ground level (10 * 25 ms = ~250 ms non blocking)
 
 static uint16_t calibrationCycles = 0;      // baro calibration = get new ground pressure value
+static uint16_t cycleCount = 0;
```

Comment:
```suggestion
static uint16_t calibrationCycleCount = 0;
```

Reply:
cycleCount also needs to be changed in other places in the file.

Reply:
Sure, that's a given consequence. But haslinghuis is experienced enough to know that without me marking every occurence 🙂

Reply:
Replaced.